DataManagementSystem/Client/CmdDirCompletion/DirectoryCompletion.py

```diff
@@ -84,7 +84,7 @@ def get_filename(self, path, dirname):
 
 
 if __name__ == "__main__":
-  from AbstractFileSystem import UnixLikeFileSystem
+  from AbstractFileSystem import UnixLikeFileSystem  # pylint: disable=import-error
```

Comment:
shouldn't this be
```suggestion
  from .AbstractFileSystem import UnixLikeFileSystem
```
I.e. explicitly importing from the neighbor module AbstractFileSystem?

Reply:
No as it's called as a script with something like `python DataManagementSystem/Client/CmdDirCompletion/DirectoryCompletion.py`. Relative imports only work inside modules and `pylint` isn't smart enough to figure it out. The same applies to `config` and `make_dfc_plots_lib` in the tests.

```bash
> ls -R
my-folder-1

./my-folder-1:
my-folder-2

./my-folder-1/my-folder-2:
A.py	B.py	C.py
> cat my-folder-1/my-folder-2/A.py
from __future__ import absolute_import
print("Import A")
import B

> cat my-folder-1/my-folder-2/B.py
from __future__ import absolute_import
print("Import B")
import B

> python2.7 my-folder-1/my-folder-2/A.py
Import A
Import B
> python3 my-folder-1/my-folder-2/A.py
Import A
Import B
> cat my-folder-1/my-folder-2/C.py
from __future__ import absolute_import
print("Import C")
from . import B

> python2.7 my-folder-1/my-folder-2/C.py
Import C
Traceback (most recent call last):
  File "my-folder-1/my-folder-2/C.py", line 3, in <module>
    from . import B
ValueError: Attempted relative import in non-package
```

Reply:
Yes, I see.
Thanks for the detailed explanation!